front/style/layout/_buttons.scss

```diff
@@ -444,9 +444,11 @@ input {
     }
 }
 
-button.btn {
-    display: inline-block;
-    padding: 10px 15px 9px;
+button {
+    .btn {
```

Comment:
Why change this? This is used in a lot of places.

Reply:
As we said, I just removed this piece of code.
💥